src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt

```diff
@@ -162,21 +162,23 @@ open class SipMessageHandler : AbstractVerticle() {
     }
 
     open fun calculateSipMessageMetrics(prefix: String, packet: Packet, message: SIPMessage) {
-        val attributes = (packet.attributes?.toMutableMap() ?: mutableMapOf()).apply {
-            packet.srcAddr.host?.let { put(Attributes.src_host, it) }
-            packet.dstAddr.host?.let { put(Attributes.dst_host, it) }
-            message.statusCode()?.let {
-                put("status_type", "${it / 100}xx")
-                put("status_code", it)
+        val attributes = (packet.attributes?.toMutableMap() ?: mutableMapOf())
```

Comment:
Packet attributes is already MutableMap. So you can do something like `packet.attributes ?: mutableMapOf()` instead